vm/kvm-self-tests/runtest.sh

```diff
@@ -141,13 +217,29 @@ function setup
 
     check
 
+   if lsmod | grep --quiet kvm_intel; then
+     rmmod kvm_intel; rmmod kvm
+     modprobe kvm; modprobe kvm_intel
+   elif lsmod | grep --quiet kvm_amd; then
+     rmmod kvm_amd; rmmod kvm
+     modprobe kvm; modprobe kvm_amd
+   fi
     rlRun "rm -rf $TMPDIR && mkdir $TMPDIR"
+    rlRun "rm -rf ${BINDIR} && mkdir -p ${BINDIR}/x86_64"
 
     rlRun "pushd '.'"
 
+    # if running on rhel8, use python3
+    if grep --quiet "release 8." /etc/redhat-release && [ ! -x /usr/bin/python ];then
+        ln -s /usr/libexec/platform-python /usr/bin/python
+    fi
+
     rlRun "cd $TMPDIR"
-    rlFetchSrcForInstalled $pkg
-    typeset rpmfile=$(ls -1 $TMPDIR/${pkg}*.src.rpm)
+    dnf download ${pkg} --source
```

Comment:
Is dnf present on older releases? In other tests, we check if it's yum or dnf that's present and use that for installations

Reply:
Great suggestion, this test is only running for RHEL8 and ARK, however I think it's a good idea to test before running the command. I'll copy the check from kpkginstall runtest.sh and try both DNF and YUM before falling back to the beakerlib.